modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,119 +1,34 @@
-----
-
-. Check the machine configs to see that the new one was added:
-+
-[source,terminal]
-----
-$ oc get MachineConfig
-----
-+
-.Example output
-[source,terminal]
-----
-NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-----
-
-. Check the nodes:
+. To enable multipathing at installation time, append the kernel arguments by entering the following command:
 +
 [source,terminal]
 ----
-$ oc get nodes
+$ coreos-installer install --append-karg rd.multipath=default --append-karg root=/dev/disk/by-label/dm-mpath-root
 ----
-+
-.Example output
-[source,terminal]
-----
-NAME                           STATUS                     ROLES    AGE   VERSION
-ip-10-0-136-161.ec2.internal   Ready                      worker   28m   v1.20.0
-ip-10-0-136-243.ec2.internal   Ready                      master   34m   v1.20.0
-ip-10-0-141-105.ec2.internal   Ready,SchedulingDisabled   worker   28m   v1.20.0
-ip-10-0-142-249.ec2.internal   Ready                      master   34m   v1.20.0
-ip-10-0-153-11.ec2.internal    Ready                      worker   28m   v1.20.0
-ip-10-0-153-150.ec2.internal   Ready                      master   34m   v1.20.0
-----
-+
-You can see that scheduling on each worker node is disabled as the change is being applied.
 
 . Check that the kernel argument worked by going to one of the worker nodes and listing
 the kernel command line arguments (in `/proc/cmdline` on the host):
```

Comment:
This is taken from the post-install multipath procedure. Is it relevant here or is there another way that the user can verify that the command worked?

Reply:
Yes, it's still relevant.